README.md

```diff
@@ -198,6 +198,7 @@ SUSHII_IMG_QUALITY=70
 
 * [sushii bot]
 * [haseul bot]
+* [hyejoo bot]
```

Comment:
If you don't have it on GitHub that's fine! Does it have a website or anything? If not, you can remove the brackets.

```suggestion
* [hyejoo bot]
* hyejoo bot
```

Reply:
can it be a top gg link?

Reply:
Yeah that's fine, you can add the link at the bottom with the other links.

Reply:
[hyejoo bot](https://top.gg/bot/733035786664149104)

is this fine? sorry, im new to pr'ing.

Reply:
[hyejoo bot]: https://top.gg/bot/733035786664149104

Reply:
It should go in the file, this is the comment so it won't modify the file.